client/client-multi/client-c/src/kaa/ext_notification_receiver.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Please, move this header to the platform folder. Rename corresponding include-s.